docs/layers/software-delivery/ecs-ecspresso/setup.mdx

```diff
@@ -14,37 +15,29 @@ import Admonition from '@theme/Admonition'
   This setup guide will help you get started with [ecspresso](https://github.com/kayac/ecspresso). It features an example app, which demonstrates how your GitHub Actions work with your infrastructure repository.
 </Intro>
 
-:::caution Default Workflows Placement
+| Steps                                                | Actions                                                                           |
+| ---------------------------------------------------- | --------------------------------------------------------------------------------- |
+| 1. Create a repository from the Example App template | [cloudposse-examples/app-on-ecs](https://github.com/cloudposse-examples/app-on-ecs) |
+| 2. Update and reapply `ecr`                          | `atmos terraform apply ecr -s core-use1-artifacts`                                |
+| 3. Validate the environment configuration            | Click Ops                                                                         |
+| 4. Create a GitHub PAT                               | Click Ops                                                                         |
+| 5. Set all Example App repository secrets            | Click Ops                                                                         |
+| 6. Deploy the example ECS services                   | `atmos workflow deploy/app-on-ecs -f app-on-ecs`                                  |
+
+<Note title="note">
 
 **All Workflows live in the example app repository**
 
 This is not a D.R.Y. setup. Workflows with `ecspresso` and `workflow` **prefixes** should be moved to a shared workflow
 repository that can be used by the rest of your organization.
```

Comment:
I don't like this wording as it sounds more like a recommendation that all workflows should live in the repository

Reply:
updated to this:

> **All Workflows live in the example app repository**
> 
> The example is not a D.R.Y. setup. Workflows with `ecspresso` and `workflow` **prefixes** should be moved to a shared workflow repository that can be used by the rest of your organization. For the sake of the example we place all workflows inside the example app to make it easier to demonstrate initial setup.

Reply:
replaced with your next comment